addition of kinetic simulation related parameters and catalytic properties

sofia-angeli opened the pull request.

sofia-angeli committed:
Add files via upload

dalito commented:
Good morning, Sofia!

The pipeline looks for the `voc4cat.xlsx` file only in the folder `/inbox-excel-vocabs` but you dropped it into the root directory. You can just move it to `/inbox-excel-vocabs` and commit the change. Be sure to edit [your PR branch](https://github.com/sofia-angeli/voc4cat/tree/voc4Cat_SA1).

Could you maybe change the title of the PR to something more meaningful (e.g. "Add concepts for kinetic modelling") and provide a short description of the addition?

sofia-angeli committed:
Add files via upload

sofia-angeli committed:
CI: vocabulary update in https://github.com/nfdi4cat/voc4cat/actions/runs/14439842921

sofia-angeli committed:
Add files via upload

sofia-angeli committed:
Delete voc4cat.xlsx

sofia-angeli committed:
CI: vocabulary update in https://github.com/nfdi4cat/voc4cat/actions/runs/14439998262

dalito commented:
This looks good now! No need to close/re-open. ...or are you trying to achieve something special?

sofia-angeli commented:
Good morning David!
I am new to github, so that explains my actions. I couldn't find the way to update the pull request after the changes I main in my branch or create a new one, so I closed it and reopened. Obviously not necessary. :blush:

dalito commented:
This pull request is still only a branch in your forked repository (it was not copied or send to voc4cat). So when you commit changes to the branch "voc4Cat_SA1" in your fork, this pull request shows the changes automatically here (because it is the same). 
Only when the pull request is merged, your code becomes part of of voc4cat.

The distributed nature of git with many new terms can be overwhelming. If you have questions , just ask.

dalito reviewed (changes requested):
See last comment for required changes.

dalito committed:
Move concepts to subdir

dalito committed:
Update RDF to v1.0 format

dalito committed:
CI: vocabulary update in https://github.com/nfdi4cat/voc4cat/actions/runs/20642901832

nmoust commented:
Dear @sofia-angeli please find below a few comments from my side. As there are some important changes made to Voc4Cat since you contributed, you should download the new version of the vocabulary xlsx. Also you should structure your concepts under Voc4Cat's top-level concepts.

| URI  | Concept                              | Definition                                                                                                                                 | Comments                                                                                                                                                                                                 |
|------|--------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| 7371 | Catalyst properties                  | A group of physical quantities and chemical components that describe the catalyst.                                                        | Change preferred label to singular (Catalyst property). Change the definition appropriately.                                                                                                           |
| 7372 | Active metal                         | The metallic material that is catalytically active for a specific reaction.                                                               | Probably not needed to be added as a concept as “Active site“, “metal oxide“, “metallic substrate“ already exist.                                                                                      |
| 7373 | Catalyst support                     | An oxide(s) with high specific surface area where active metal is dispersed.                                                              | Change definition to be more general. There is a concept “Supported catalyst”.                                                                                                                          |
| 7374 | Metal loading                        | The weight of active metal over the total weight of the catalyst.                                                                         | Keep it more general? E.g., loading.                                                                                                                                                                     |
| 7375 | Particles mean diameter              | The mean diameter of the catalyst particles in powder form.                                                                               |                                                                                                                                                                                                          |
| 7376 | Fixed bed reactor loading            | The mass of catalyst loaded in a fixed bed reactor.                                                                                       | There is a concept “Catalyst bed volume”. Maybe the concept name can be changed to “Catalyst bed mass”.                                                                                                  |
| 7377 | Hydrogen uptake                      | The amount of adsorbed hydrogen on the catalytic surface during temperature programmed desorption measurement for the determination of the metal dispersion. | I suggest that you create a more general “uptake” concept with a more “general” definition. I am thinking that having two separate concepts for “hydrogen uptake” and “carbon monoxide uptake” is needed. |
| 7378 | Hydrogen temperature programmed desorption | A technique which is used for the determination of the dispersion of the active metal on the catalyst.                                   | I suggest that you create a “temperature programmed desorption” concept and then (if needed) have the “hydrogen temperature programmed desorption” concept.                                              |
| 7379 | Carbon monoxide uptake               | The amount of adsorbed carbon monoxide on the catalytic surface during temperature programmed desorption measurement for the determination of the metal dispersion. | Check comment for “hydrogen uptake”.                                                                                                                                                                      |
| 7380 | Metal nanoparticle size              | The size / diameter of the metal nanoparticles dispersed on a porous support.                                                            | I think that this is similar to the concept (7375) “particles mean diameter” of yours.                                                                                                                  |
| 7381 | Metal dispersion                     | The mole fraction of metal on the surface with respect to overall moles of metal dispersed on the catalyst support.                      | There is a more general concept (7044) “dispersion”.                                                                                                                                                     |
| 7382 | Metal specific surface area          | The surface area covered by dispersed metal over the mass of catalyst.                                                                    | There is a more general concept (0013) “specific surface area”.                                                                                                                                          |
| 7383 | Area to volume ratio                 | The ration of the surface area of the catalyst divided by the volume of the catalytic reactor.                                           | There is a typo in the definition (ration → ratio).                                                                                                                                                      |
| 7384 | Reactor wall thickness               | The thickness of the wall of a chemical reactor.                                                                                          | For both 7384 and 7385 the definition is not precise enough. For reactor wall thickness it might be better to define it as the vessel that the reactants are in contact with (just a suggestion).          |
| 7385 | Reactor wall temperature             | The temperature of the reactor wall.                                                                                                      | Similar to the comment for 7384. The definition should be more precise, e.g., where the temperature is measured.                                                                                        |
| 7386 | Isothermal reactor operation         | The mode of the reactor operation at constant temperature.                                                                                |                                                                                                                                                                                                          |
| 7387 | Adiabatic reactor operation          | The mode of reactor operation where no heat exchange between the reactants and its surroundings is permitted via the reactor wall.       |                                                                                                                                                                                                          |

sofia-angeli commented:
Dear Niko and David,
Please find below some notes on my revisions.<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/td5696/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/td5696/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>
<!--table
	{mso-displayed-decimal-separator:"\.";
	mso-displayed-thousand-separator:"\,";}
@page
	{margin:.75in .7in .75in .7in;
	mso-header-margin:.3in;
	mso-footer-margin:.3in;}
tr
	{mso-height-source:auto;}
col
	{mso-width-source:auto;}
br
	{mso-data-placement:same-cell;}
td
	{padding-top:1px;
	padding-right:1px;
	padding-left:1px;
	mso-ignore:padding;
	color:black;
	font-size:11.0pt;
	font-weight:400;
	font-style:normal;
	text-decoration:none;
	font-family:Calibri, sans-serif;
	mso-font-charset:0;
	mso-number-format:General;
	text-align:general;
	vertical-align:bottom;
	border:none;
	mso-background-source:auto;
	mso-pattern:auto;
	mso-protection:locked visible;
	white-space:nowrap;
	mso-rotate:0;}
.xl63
	{color:#1F2328;
	font-size:8.0pt;
	font-family:"Segoe UI", sans-serif;
	mso-font-charset:0;
	text-align:left;
	vertical-align:middle;
	background:#F6F8FA;
	mso-pattern:black none;
	white-space:normal;
	padding-left:21px;
	mso-char-indent-count:1;}
.xl64
	{color:#1F2328;
	font-size:8.0pt;
	font-family:"Segoe UI", sans-serif;
	mso-font-charset:0;
	text-align:left;
	vertical-align:middle;
	background:white;
	mso-pattern:black none;
	white-space:normal;
	padding-left:21px;
	mso-char-indent-count:1;}
-->
</style>
</head>

<body link="#0563C1" vlink="#954F72">


7374 | Metal loading | The weight of active metal over   the total weight of the catalyst. | Keep it more general? E.g.,   loading. | then how would you differentiate it from catalyst loading e.g. on a monolithic structure or the loading of the catalyst in a reactor? I would rename it to active phase loading instead of   metal because in some cases the active phase is not metallic. But I will add the metal to the alternative labels
-- | -- | -- | -- | --
7377 | Hydrogen uptake | The amount of adsorbed hydrogen on   the catalytic surface during temperature programmed desorption measurement   for the determination of the metal dispersion. | I suggest that you create a more   general “uptake” concept with a more “general” definition. I am thinking that   having two separate concepts for “hydrogen uptake” and “carbon monoxide   uptake” is needed. | I created a parent concept named   as : uptake and used H2 and CO uptake as children.
7380 | Metal nanoparticle size | The size / diameter of the metal   nanoparticles dispersed on a porous support. | I think that this is similar to   the concept (7375) “particles mean diameter” of yours. | In the 7375 concept, the   particle size refers to the particles of the catalyst powder, while this   concept refers to the size of the metal nonaparticles dispersed on a support.   Eg a catalyst with 250 microns particle size (powder) can have dispersed   metal nanoparticles of 15nm.
381 | Metal dispersion | The mole fraction of metal on the   surface with respect to overall moles of metal dispersed on the catalyst   support. | There is a more general concept   (7044) “dispersion”. | concept removed and added a   altenative label to dispersion
7382 | Metal specific surface area | The surface area covered by   dispersed metal over the mass of catalyst. | There is a more general concept   (0013) “specific surface area”. | concept removed and added a   altenative label to dispersion



</body>

</html>

sofia-angeli commented:
[voc4cat.xlsx](https://github.com/user-attachments/files/26298929/voc4cat.xlsx)
